lib/embulk/guess/jsonl.rb

```diff
@@ -20,8 +20,15 @@ def guess_lines(config, sample_lines)
 
         return {} if rows.size <= 3
 
-        columns = Embulk::Guess::SchemaGuess.from_hash_records(rows).map do |c|
-          column = {name: c.name, type: c.type}
+        array_of_hash = Array(rows)
```

Comment:
It seems that `rows` is an Array and this Array function makes no sense.

Reply:
fixed